packages/core/src/result-summary.ts

```diff
@@ -429,18 +467,124 @@ class Notification {
   description: string
   severity: string
   position: NotificationPosition | {}
+  severityLevel: NotificationSeverityLevel
+  category: NotificationCategory
+  rawSeverityLevel: string
+  rawCategory?: string
```

Comment:
Severity level was already in the object and it will return by older protocol version, see Notification#severity. However, the category will not be return by older server versions.